docs/releases.md

```diff
@@ -0,0 +1,123 @@
+
+# Release Documentation
+
+## Abstract
+
+In order to provide a generic guidance to users, downstream maintainers and
+any other consumers of the Cloud Hypervisor project, this document
+describes the release process, release cadence, stability expectations and
+related topics.
+
+## Basic Terms
+
+### Stability
+
+In the context of Cloud Hypervisor, the points of reference regarding stability
+are as below:
+
+- [REST API](api.md#rest-api)
+- [Command line options](api.md#command-line-interface)
+- [Device Model](device_model.md)
+- Device tree, device list, ACPI, Hyper-V enlightenments, UEFI and any other
+  features exposed to guest
+- KVM compatibility
+- Rust edition compatibility
+
+This list is incomplete but this document serves as a best effort guide to stability
+across releases.
+
+### Experimental features
+
+Features that are being actively developed and are possibly incomplete or known to be
+unstable are considered experimental. These features are in no case recommended for
+use in production.
+
+Features currently considered experimental:
+
+- TDX
+- vfio-user
+- vDPA
+
+### Security
+
+Security issues usually involve only defects directly related to Cloud Hypervisor.
```

Comment:
This is not true. If a dependency has an issue a new Cloud Hypervisor release needs to be made since all dependencies are statically linked (except libc)

Reply:
Yep, I'm only underlining that "security issue in dependency" != "security issue in cloud hypervisor itself". Also a security issue in a dependency, that doesn't affect CH itself, won't require a new release.  

Thanks

Reply:
OK, moved this out as well. The intention here was actually same - to answer some question beforehand. Perhaps there's no need to expose this kind of detail level anyway. 

Thanks